app/src/test/java/com/teamagam/gimelgimel/app/integration/interactors/DisplayVectorLayerInteractorTest.java

```diff
@@ -148,11 +148,16 @@ private Scheduler createTestScheduler() {
     }
 
     private VectorLayer createVectorLayer1() {
```

Comment:
I liked that refactor, but now I find no good reason for these two methods to exist. I think both should be inlined.